scripts/js_ts_linter.py

```diff
@@ -0,0 +1,1065 @@
+                        angular_constants_list.append(angular_constant_name)
+
+                    angular_constants_set = set(angular_constants_list)
+                    if len(angular_constants_set) != len(
+                            angular_constants_list):
+                        failed = True
+                        python_utils.PRINT(
+                            '%s --> Duplicate constant declaration found.'
+                            % filepath)
+                    if corresponding_angularjs_filepath in (
+                            angularjs_source_filepaths_to_constants_dict):
+                        angular_minus_angularjs_constants = (
+                            angular_constants_set.difference(
+                                angularjs_source_filepaths_to_constants_dict[
+                                    corresponding_angularjs_filepath]))
+                        for constant in angular_minus_angularjs_constants:
+                            failed = True
+                            python_utils.PRINT(
+                                '%s --> The constant %s is not declared '
+                                'in the corresponding angularjs '
+                                'constants file.' % (filepath, constant))
+
+            if failed:
+                summary_message = (
+                    '%s  Constants declaration check failed, '
+                    'see messages above for constants with errors.' % (
+                        _MESSAGE_TYPE_FAILED))
+            else:
+                summary_message = '%s  Constants declaration check passed' % (
+                    _MESSAGE_TYPE_SUCCESS)
+            summary_messages.append(summary_message)
+            python_utils.PRINT(summary_message)
+
+        return summary_messages
+
+    def _check_dependencies(self):
+        """Check the dependencies related issues. This runs
+        _check_sorted_dependencies and
+        _match_line_breaks_in_controller_dependencies
+        in parallel.
+        """
+        methods = [
+            self._check_sorted_dependencies,
+            self._match_line_breaks_in_controller_dependencies
+        ]
+        super(JsTsLintChecksManager, self)._run_multiple_checks(*methods)
+
+    def perform_all_lint_checks(self):
+        """Perform all the lint checks and returns the messages returned by all
+        the checks.
+
+        Returns:
+            all_messages: str. All the messages returned by the lint checks.
+        """
+
+        if not self.all_filepaths:
+            python_utils.PRINT('')
+            python_utils.PRINT(
+                'There are no JavaScript or Typescript files to lint.')
+            return []
+
+        self.parsed_js_and_ts_files = self._validate_and_parse_js_and_ts_files()
+        self.parsed_expressions_in_files = (
+            self._get_expressions_from_parsed_script())
+
+        common_messages = super(
+            JsTsLintChecksManager, self).perform_all_lint_checks()
```

Comment:
Is this going to kick off the processes?

Reply:
Removed.